pkg/storage/stores/shipper/bloomshipper/client.go

```diff
@@ -33,20 +33,28 @@ type Ref struct {
 	TenantID                       string
 	TableName                      string
 	MinFingerprint, MaxFingerprint uint64
-	StartTimestamp, EndTimestamp   int64
+	StartTimestamp, EndTimestamp   int64 // must be millisecond precision timestamp aka model.Time
```

Comment:
It would make even more sense to use `model.Time` type.

Reply:
I agree 👍